feature-pack/src/main/resources/content/appclient/configuration/appclient.xml

```diff
@@ -154,6 +154,13 @@
                         <policy-module code="Client" flag="required"/>
                     </authorization>
                 </security-domain>
+                <security-domain name="wise-security-domain" cache-type="default">
+                    <authentication>
+                        <login-module code="RealmDirect" flag="required">
+                            <module-option name="realm" value="ManagementRealm"/>
+                        </login-module>
+                    </authentication>
+                </security-domain>
```

Comment:
Just to be clear, I should move this into ./security/subsystem/src/main/resources/subsystem-templates/security.xml
Right?

----- Original Message -----
> From: "Tomaz Cerar" <notifications@github.com>
> To: "wildfly/wildfly" <wildfly@noreply.github.com>
> Cc: "rsearls" <rsearls@redhat.com>, "Author" <author@noreply.github.com>
> Sent: Tuesday, July 5, 2016 5:53:11 AM
> Subject: Re: [wildfly/wildfly] [WFLY-6779] added WISE feature code (#8998)
> 
> > @@ -154,6 +154,13 @@
> >                          <policy-module code="Client" flag="required"/>
> >                      </authorization>
> >                  </security-domain>
> > +                <security-domain name="wise-security-domain"
> > cache-type="default">
> > +                    <authentication>
> > +                        <login-module code="RealmDirect" flag="required">
> > +                            <module-option name="realm"
> > value="ManagementRealm"/>
> > +                        </login-module>
> > +                    </authentication>
> > +                </security-domain>
> 
> app client needs this? really?
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/wildfly/wildfly/pull/8998/files/1cc1e6d45243b24d555028c72475d8565505c969#r69535902